yarn-project/aztec-nr/slow-updates-tree/src/slow_map.nr

```diff
@@ -0,0 +1,253 @@
+use dep::aztec::context::{PrivateContext, PublicContext, Context};
+use dep::aztec::state_vars::public_state::PublicState;
+use dep::aztec::types::type_serialization::TypeSerializationInterface;
+use dep::aztec::oracle::storage::{storage_read, storage_write};
+
+use dep::std::option::Option;
+
+use dep::std::merkle::compute_merkle_root;
+
+// The epoch lenght is just a random number for now.
+global EPOCH_LENGTH: u120 = 100;
+
+fn compute_next_change(time: Field) -> Field {
+  (((time as u120 / EPOCH_LENGTH + 1) * EPOCH_LENGTH)) as Field
```

Comment:
Why is there the +1 in the denominator? Is it to prevent some rounding issue?

Reply:
It is not in the denominator, it is just looking like garbage hehe, it is practically
```
((time / EPOCH_LENGTH) + 1) * EPOCH_LENGTH
```
So it is just the time of the "next" epoch.